app/components/shared/inputs/index.ts

```diff
@@ -41,6 +41,7 @@ export interface INumberMetadata extends IInputMetadata {
   min?: number;
   max?: number;
   placeholder?: string;
+  isInteger?: boolean;
```

Comment:
If you're going to use the same input for `int` and `float`, what do you think about to have a bit more flexible option, something like `precision`?
```
1 // precision = 0
1.2 // precision = 1
1.23 // precision = 2
1.12345... // precision = -1
```

Reply:
i briefly considered that but i don't think we actually have validations on precision anywhere in the code base, but more specifically i didn't want to have to handle all currently roundings for every currency, which is the most likely case where we would use precision